packages/block-editor/src/store/selectors.js

```diff
@@ -130,6 +132,7 @@ export const getBlock = createSelector(
 			...block,
 			attributes: getBlockAttributes( state, clientId ),
 			innerBlocks: getBlocks( state, clientId ),
+			parent,
```

Comment:
@youknowriad - can you perform sanity check here?

Reply:
@gziolo I didn't notice that before (perhaps because it was a part of PR on Floating Toolbar) but it seems the `parent` prop returned here from `getBlocks` function is not used anywhere. On the second hand there is also `getBlockRootClientId` which return the parent of passed `clientId`. I think we can just get rid of all changes in this `selectors.js` file and we'll be fine .

Reply:
> I think we can just get rid of all changes in this `selectors.js` file and we'll be fine .

I was mostly concern about performance implications. In the past, `getBlocks` was one of the top reasons which triggered unnecessary re-renders. It's probably safer to leave it as it is then.

Reply:
adding `parent` inside `getBlocks` selector will be removed from FloatingToolbar PR which this branch based.